modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -455,6 +452,11 @@ This setting can be established and retrieved either for an individual replicati
 | JSON Object
 | Each of the key-value pairs in the JSON object should map a collection-specifier (in the form `scope.collection`) to the name of a defined merge-function.
 
+| `mobile`
+| Active or Off
+| Default: `Off`.
+When set to `Active`, enables the setting _XDCR Active-Active with Sync Gateway 4.0+_ on the clusters of both sides of the replication. The default value `Off` indicates that the replication setup supports either _XDCR Active-Passive with Sync Gateway_ or _XDCR Active-Active without Sync Gateway_. For more information, see xref:learn:clusters-and-availability/xdcr-active-active-sgw.adoc[XDCR Active-Active with Sync Gateway].
+
```

Comment:
This entry is fine.

A few lines down -- lines 469 to 474, there is the optimisticReplicationThreshold.

For the optimisticReplicationThreshold, we should add the below sentence like you did for the optimistic replication info in the advanced settings page:

XDCR optimistic replication is applicable only when the `enableCrossClusterVersioning` property is disabled.